### PR TITLE
Multiple Destinations 

### PR DESCRIPTION
**Pull request by okie-creator**
adding multiple destinations to getNotam

### CODE REVIEW

File: GetNOTAM.py [hunk 1]
```diff
@@ -75,24 +79,38 @@ def buildNotam(effectiveStartDate, effectiveEndDate, long, lat, combined_core_no
...
+        print(fLong)
+        print(fLat)
+        
+        combined_core_notam_data = buildNotam(effectiveStartDate, effectiveEndDate, airLocation[1], airLocation[0], combined_core_notam_data)
         print(len(combined_core_notam_data))
+        wlong = float(long)
+        wlat = float(lat)
+        bearing = MinimalCirclesPath.calculateBearing(lat,long,fLat, fLong)
+    
+        while (bearing >= 0 and wlong >= fLong and wlat <= fLat) or \
```
Comment (Gabrieldowen) [inline GetNOTAM.py:104]: I definitely think some comments would be useful for each of these blocks/loops

File: TestData/TestNOTAM.json
Comment (Gabrieldowen) [inline TestData/TestNOTAM.json:1]: we should also make sure this json and these binary files are in the git ignore so we are not adding unnecessary files in each commit

File: GetNOTAM.py (hunk 1 above)
Comment (Gabrieldowen) [reply GetNOTAM.py:104]: also I am not sure what wlong/wlat is supposed to represent

File: GetNOTAM.py [hunk 2]
```diff
@@ -17,19 +18,24 @@
...
-    airLocation = AirportsLatLongConverter.get_lat_and_lon(location)
-    finalAirLocation = AirportsLatLongConverter.get_lat_and_lon(finalLocation)
-
+    locations.append(input("Enter airport: "))
+    inputF = input("Would you like to have multiple end locations (Y/N)")
+    if inputF == "N":
+        locations.append(input("Enter ending airport: "))
+    else:      
+        location = input("Enter airport(Type N to stop): ")
+        while(location.upper() != "N"):
```
Comment (n-grounds) [inline GetNOTAM.py:30]: a "sentinel" value like `"N"` is fine, but I'd also suggest (from a usability standpoint) to allow a empty string to be a "sentinel" value to terminate the loop

File: GetNOTAM.py [hunk 3]
```diff
@@ -75,24 +79,38 @@ def buildNotam(effectiveStartDate, effectiveEndDate, long, lat, combined_core_no
...
-        print(wlat)
-        print(wlong)
+    for i, location in enumerate(locations[:-1]):
+        print(location)
+        airLocation = AirportsLatLongConverter.get_lat_and_lon(location)
+        finalAirLocation = AirportsLatLongConverter.get_lat_and_lon(locations[i+1])
+        long = airLocation[1]
+        lat = airLocation[0]
+        print(lat)
+        print(long)
```
Comment (n-grounds) [inline GetNOTAM.py:92]: Presumably these `prints` here and below (and above) are for testing or debugging purposes.  Are we planning to remove them before merging?  If not, can we at least convert them to something more helpful, e.g.,
```python
print( f"Converted airport {location} into latitude, longitude of {lat}, {long}" )
```

**Pull request closed without merging**